webclient/src/i18n.yaml

```diff
@@ -6,14 +6,14 @@ search:
   results: {de: "Ergebnisse", en: "results"}
   approx: {de: "ca.", en: "approx."}
 footer:
-  contact:
-    text: {de: "Kontakt", en: "Contact"}
-    link: {de: "kontakt", en: "contact"}
+  sourcecode:
+    text: {de: "Source Code", en: "Source Code"}
   privacy:
     text: {de: "Datenschutz", en: "Privacy"}
-    link: {de: "datenschutz", en: "privacy"}
+    link: {de: "datenschutz", en: "datenschutz"}
```

Comment:
Before going online, we should translate the page to English. That however can be done later in another PR.

Reply:
I added a short notice that the privacy terms are not you available in English. We need in any case a remark that only the German version is legally binding